bigbluebutton-html5/imports/startup/client/auth.js

```diff
@@ -67,6 +66,21 @@ export function joinRouteHandler(nextState, replace, callback) {
 
           const handledHTML5Parameters = [
```

Comment:
I still think that the server should be responsible to which parameters we want to handle.